recipes/_windows.rb

```diff
@@ -35,7 +51,7 @@
 end
 
 package "Sensu" do
-  source "#{node['sensu']['msi_repo_url']}/sensu-#{node['sensu']['version']}.msi"
+  source "#{node['sensu']['msi_repo_url']}/sensu-#{node['sensu']['version']}-#{kernel}.msi"
```

Comment:
I think this should be `"#{node['sensu']['msi_repo_url']}/#{version}/sensu-#{node['sensu']['version']}-#{kernel}.msi"`

Reply:
Can do, but it would mean that the OS version is always part of the url.

If we keep it like this, anybody can overwrite the #{node['sensu']['msi_repo_url']} attribute and point it to a different url also without the OS version bound in the url itself.
For example, I cannot overwrite the attribute anymore with www.google.com/download/ if the client is located at www.google.com/download/sensu-client-x64.msi. I then have to save the file in www.google.com/download//sensu-client-x64.msi, for each OS that I have 1 time.

A reason to save/cache the binary somewhere else can be, if the original url changes multiple times or the installed version is removed from the original url.

Reply:
Fair point but that's true about the kernel version as well, they could rename the file because they feel they don't need the `(x86|x|64)` version. I feel like if they want to use  a different (internal?) location they should mirror the repo structure and copy several versions over (for rollback) not just copy the one binary they want. By making it single attribute it is harder to override for just a particular instance. By just needing to give it the base path and version it should figure out the platform and pull from the right location. In my experience most shops do not have all servers with the exact same OS version especially in windows land as things tend to be more manual and therefore servers are often left on older versions of windows server while newer systems are put on newer versions.

Reply:
True that. 

We use an internal artifact system. Can't remember the exact reason. It's being used by ~20 customers of us. 

I updated the recipe/attributes files. I already had it ready :-) 

When I bump up the cookbook I will also need to update the Artifacts repo, but that's not a big issue.